frontend/packages/vkt/src/interfaces/clerkExamEvent.ts

```diff
@@ -0,0 +1,61 @@
+import { Dayjs } from 'dayjs';
+
+import { EnrollmentStatus, ExamLanguage, ExamLevel } from 'enums/app';
+import { WithId, WithVersion } from 'interfaces/with';
+
+interface Person {
+  identityNumber: string;
+  lastName: string;
+  firstName: string;
+}
+
+export interface ClerkExamEventEnrollmentResponse
+  extends Partial<WithId>,
+    Partial<WithVersion>,
+    Omit<Enrollment, 'previousEnrollmentDate'> {
+  previousEnrollmentDate: string;
+}
+
+export interface Enrollment {
+  person: Person;
+  oralSkill: boolean;
+  textualSkill: boolean;
+  understandingSkill: boolean;
+  speakingPartialExam: boolean;
+  speechComprehensionPartialExam: boolean;
+  writingPartialExam: boolean;
+  readingComprehensionPartialExam: boolean;
+  status: EnrollmentStatus;
+  previousEnrollmentDate: Dayjs;
+  digitalCertificateConsent: boolean;
+  email: string;
+  phoneNumber: string;
+  street?: string;
+  postalCode?: string;
+  town?: string;
+  country?: string;
+}
+
+export interface ClerkExamEventBasicInformation {
+  language: Exclude<ExamLanguage, 'ALL'>;
```

Comment:
`ALL` lienee lähinnä käyttöliittymän filttereitä varten luotu arvo? Jos näin, niin voisi periaatteessa olla järkevää ottaa se ulos `ExamLanguage`-enumista.

Reply:
Tuossa on se etu, että käyttöliittymäpuolella voi viitata samaan enumiin ja tsekata languageFiltteröinnissä sitten esim. `examEvent === ExamLanguage.FI`.

Jos käyttöliittymälle määrittää uuden enumin niin tuossa filtteröinttissä joutuu sitten hieman rumannäköisesti käyttään:
`
filteredExamEvents = filteredExamEvents.filter(
  (e) => e.language === (languageFilter as unknown as ExamLanguage)
);
`
Interface-puolella toki tää nykyinen malli hieman bloattaa kun tarvii sitten käyttää datan tyypityksessä: `Exclude<ExamLanguage, 'ALL'>;`

Huikkaa toki jos missaan jonkin selkeän ratkaisun tässä ajatuksenjuoksussa.